packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -396,6 +411,7 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
 
     public process(messageArg: ISequencedDocumentMessage, local: boolean, localOpMetadata: unknown): void {
         this.verifyNotClosed();
+        this.verifyNotTombstoned();
```

Comment:
There should be just one method that does this. Having 2 creates the chance of someone accidentally leaving one out. Maybe rename verifyNotClosed to verifyValid or something to that effect. It accepts a parameter that says check for tombstone or not with clear description of when to use this flag.